src/components/admin-complaints/index.js

```diff
@@ -51,6 +51,17 @@ const types = [
   { key: 'cle', text: 'CLEANING', value: 'cle' },
 ]
 
+const entries = [
+  { key: '5', text: '5', value: '5' },
+  { key: '10', text: '10', value: '10' },
+  { key: '15', text: '15', value: '15' },
+  { key: '20', text: '20', value: '20' },
+  { key: '25', text: '25', value: '25' },
+  { key: '50', text: '50', value: '50' },
+  { key: '100', text: '100', value: '100' },
+  { key: '150', text: '150', value: '150' },
+]
+
```

Comment:
Create a constants file inside /components itself and define this entries array there to avoid code repetition.

Reply:
Done